indico/migrations/versions/20210215_1052_f26c201c8254_add_complete_registration_attach_ical_.py

```diff
@@ -0,0 +1,25 @@
+"""Add complete_registration_attach_ical to RegistrationForm
+
+Revision ID: f26c201c8254
+Revises: 26985db8ed12
+Create Date: 2021-02-15 10:52:15.353452
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = 'f26c201c8254'
+down_revision = '26985db8ed12'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column('forms', sa.Column('complete_registration_attach_ical', sa.Boolean(), nullable=False, server_default='false'), schema='event_registration')
```

Comment:
While we explicitly disabled the line length limit check in flake8 for alembic revisions, the goal is to not get warnings when having raw SQL in there - let's still keep max 120 chars per line for anything else.